src/libs/runtime-utils.js

```diff
@@ -69,6 +69,11 @@ export const runtimeCost = ({ runtimeConfig, status }) => {
   }
 }
 
+export const getGalaxyCost = app => {
+  return hourlyKubernetesAppCost(app || { kubernetesRuntimeConfig: { machineType: 'n1-standard-8' } }) +
```

Comment:
its probably desirable to pull this machineType off the app itself at some point, since the API GET does specify this, and it can be passed in at creation time

Reply:
oh good call - I think when this logic was added the response did not include it? I will make the update now though

Reply:
oh, i think it actually is doing that, its just appending that kubernetesRuntimeConfig if the app doesn't have it? Still probably good to update it, the app should always have it

Reply:
oh I mistook where this was, this line is only if there is no app already then we are setting the default to be a 'n1-standard-8' machine. If there is an app then we get the machine type off the app